Campus Density/DiningCells/GraphCell.swift

```diff
@@ -63,6 +62,11 @@ class GraphCell: UICollectionViewCell, UIGestureRecognizerDelegate {
         fatalError("init(coder:) has not been implemented")
     }
 
+    /// Returns the text description of density at a certain hour
+    static func descriptionLabelText(selectedHour: Int, densityMap: [Int: Double]) -> String {
```

Comment:
Does this need to be a `static` function?

Reply:
It's called in `PlacesDetailViewController+Extension` instead of repeating the same format string manually, but I'm not sure about best practices regarding where this function could be located -- I'll look into it a bit further